doc/man1/openssl-verification-options.pod

```diff
@@ -151,20 +173,21 @@ The first step is to check that each certificate is well-formed.
 Part of these checks are enabled only if the B<-x509_strict> option is given.
 
 The second step is to check the extensions of every untrusted certificate
-for consistency with the supplied purpose.
-If the B<-purpose> option is not given then no such checks are done
-except for SSL/TLS connection setup,
-where by default C<sslserver> or C<sslclient>, are checked.
-The target or "leaf" certificate, as well as any other untrusted certificates,
-must have extensions compatible with the specified purpose.
+for consistency with any given purpose.
+If the B<-purpose> option is not given
+then no such checks are done except for SSL/TLS connection setup,
+where by default C<sslserver> or C<sslclient> is checked.
+Each certificate must either not have an EKU extension,
+or the EKU extension must include B<anyExtendedKeyUsage>
+or the given required purpose.
```

Comment:
`s/or/for/`?

Reply:
Actually something else was meant here, namely 
```
Each certificate must either not have an EKU extension,
or the EKU extension must include the special value B<anyExtendedKeyUsage>
or the EKU extension must include an EKU associated the given required purpose.
```
yet it turns out that things are much more complicated.

I just noticed that a few lines below there was an outdated link, which I corrected, to a section further down in the same document that was already covering the topic in much more detail.
So I removed again the newly added sentence here and updated and strengthened the explanation on EKUs  etc. in that section instead.